test/parallel/test-dgram-send-callback-buffer-length-empty-address.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+const dgram = require('dgram');
+const client = dgram.createSocket('udp4');
+
+const buf = Buffer.alloc(256, 'x');
+const offset = 20;
+const len = buf.length - offset;
+
+const onMessage = common.mustCall(function messageSent(err, bytes) {
```

Comment:
FWIW, these two files are copypasta from `test-dgram-send-callback-buffer-length` which has the same issues.